.gitignore

```diff
@@ -6,6 +6,7 @@ pkg/generic/test/generated_*
 pkg/kube/unstructured/test/templates/generated_*
 kubedog
 coverage.txt
+examples/templating/generic/generic
```

Comment:
It's `generic/generic` path only or should it ignore any `generic/*` path?

Reply:
That is a binary. Its created after building the examples with the `build-examples` make target